hathor/feature_activation/feature.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from enum import Enum
+
+
+class Feature(Enum):
+    """
+    An enum containing all features that participate in the feature activation process, past or future,
+    activated or not, for all networks. Features should NOT be removed from this enum, to preserve history.
+    """
+
+    NOP_FEATURE = 0
```

Comment:
Shouldn't it have a `name` attribute? Maybe it's better than an integer.

Reply:
I changed it to a string enum in https://github.com/HathorNetwork/hathor-core/pull/611/commits/85ba2615408565af9abb032ac41992c4531da6ec

Reply:
Ok. Thanks!